tests/Test/Environment/AttributesTest.php

```diff
@@ -4,19 +4,23 @@
 
 use Fixture;
 use PHPUnit\Framework\TestCase;
+use my127\Workspace\Tests\IntegrationTestCase;
 
-class AttributesTest extends TestCase
+class AttributesTest extends IntegrationTestCase
 {
     /** @test */
     public function host_os_family_is_made_available_as_an_attribute()
     {
-        Fixture::workspace(<<<'EOD'
+        $this->workspace()->put('workspace.yml', <<<'EOD'
```

Comment:
as this is very common, could should make a utility method on the `Workspace` to avoid passing the filename.

Reply:
or perhaps better to put the specific "utility" method in the IntegrationTestCase and keep the `Workspace` utility opinionated.